### PR TITLE
Libtiff patch

### PR DESCRIPTION
**Pull request by abadawi591**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/libtiff/libtiff.spec [hunk 1]
```diff
@@ -67,6 +68,10 @@ make %{?_smp_mflags} -k check
 %{_mandir}/man3/*
 
 %changelog
+
```
Comment (PawelWMS) [inline SPECS/libtiff/libtiff.spec:71]: Nit-pick clean-up comment:
```suggestion
```

**Review by PawelWMS — APPROVED** [review]
Just one small clean-up request.

File: SPECS/libtiff/libtiff.spec (hunk 1 above)
Comment (sameluch) [reply SPECS/libtiff/libtiff.spec:81]: I Added a new entry in the change log with no space inbetween, marking this one as resolved.

**Comment by sameluch** [comment]
Successful buddy builds for libtiff with new patches:
ARM64 Buddy Build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=265877&view=results
AMD64 Buddy Build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=265871&view=results

File: SPECS/libtiff/libtiff.spec [hunk 2]
```diff
@@ -67,6 +75,17 @@ make %{?_smp_mflags} -k check
 %{_mandir}/man3/*
 
 %changelog
+* Fri Nov 18 2022 Sam Meluch <sammeluch@microsoft.com - 4.4.0-6
```
Comment (nicogbg) [inline SPECS/libtiff/libtiff.spec:78]: 4.4.0-7 ?

**Review by nicogbg — APPROVED** [review]

File: SPECS/libtiff/libtiff.spec (hunk 2 above)
Comment (sameluch) [reply SPECS/libtiff/libtiff.spec:78]: 4.4.0-6 was never committed to main, condense changelog comments instead of incrementing the release number.

**Pull request merged**